pages/linux/brew.md

```diff
@@ -10,13 +10,13 @@
 
 `brew install {{formula}}`
 
-- List installed formulae [with matching name]:
+- List installed formulae, argument `formula` is optional:
 
-`brew list {{[text]}}`
+`brew list {{formula}}`
```

Comment:
I don't think the example below is equivalent, since both options do the same operation, but ok, your phrasing sounds reasonable and makes the page self-consistent.